java/hibernate/README.md

```diff
@@ -64,5 +64,83 @@ We have generated schema definitions that are currently applied in the database
 ./flway migrate
 ```
 
+## Delete REST API request 
+
+### 1. `/users` end point
+  
+ You can use the following curl command for deleting a user:
+ 
+ ```
+ curl --data '{ "userId": “1” }' -v -X DELETE -H 'Content-Type:application/json' http://localhost:8080/users
+ ```
+
+This will return the user details which is deleted:
```

Comment:
You could mention what happens to the orders placed by a user when the user or product is deleted.